### DISCUSSION TITLE
Implement so there is no limit on how many output shift registers per Arduino pin

### DISCUSSION THREAD

**Discussion by MarkusA350** [MobiFlight/MobiFlight-FirmwareSource#288] [Ideas]
Hello everyone!
I am currently using SimVim for my home cockpit project, however would like to move over to MobiFlight. 
The primary hurdle I'm facing relates to output shift registers. I asked in the discord server and they said that you can chain up to four output shift registers, then they all share the same pins on the arduino. 
However, with SimVim, there's no limitation on the number of output shift registers per pin on the Arduino, so you could have more than 4 output shift registers per pin. 
Presently, I have 18 output shift registers with all the L's connected to one Arduino pin and all the D's to another. Rewiring everything to align with MobiFlight's requirements would be quite challenging, and requires a lot of work. Therefore, it would be greatly appreciated by myself and others transitioning from SimVim to MobiFlight if a similar flexibility could be implemented. Specifically, the ability to connect all the L's from output shift registers to one pin on the Arduino and likewise for the D's, without the constraint of being limited to four output shift registers per pin.

Grateful for this wonderful community!
I sincerely hope that the implementation of something like this becomes a reality, if feasible

/Markus

**Comment by elral** [comment]
The firmware can already handle shared pins, e.g. `Latch` and `Data` from the output shifters.

The "limitation" comes from the connector, while defining a new device only not already used pins can be defined for the pins of this new device.
